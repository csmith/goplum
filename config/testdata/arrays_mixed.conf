defaults {
    arr = [1, true, "oops"]
}
