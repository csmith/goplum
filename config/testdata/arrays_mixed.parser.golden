"unexpected boolean (true) at line 2 column 14, expecting one of: integer, end of array"